apps/web/src/components/Menu/UserMenu/hooks/useCakeBenefits.ts

```diff
@@ -7,15 +7,15 @@ import { getBalanceNumber } from '@pancakeswap/utils/formatBalance'
 import { useTranslation } from '@pancakeswap/localization'
 import { useChainCurrentBlock } from 'state/block/hooks'
 import { getVaultPosition, VaultPosition } from 'utils/cakePool'
-import { getCakeVaultAddress, getAddress } from 'utils/addressHelpers'
+import { getCakeVaultAddress } from 'utils/addressHelpers'
 import { multicallv2 } from 'utils/multicall'
 import { getActivePools } from 'utils/calls'
 import cakeVaultAbi from 'config/abi/cakeVaultV2.json'
 import { BIG_ZERO } from '@pancakeswap/utils/bigNumber'
-import { convertSharesToCake } from '../../../../views/Pools/helpers'
-import { getScores } from '../../../../views/Voting/getScores'
-import { PANCAKE_SPACE } from '../../../../views/Voting/config'
-import * as strategies from '../../../../views/Voting/strategies'
+import { convertSharesToCake } from 'views/Pools/helpers'
+import { getScores } from 'views/Voting/getScores'
+import { PANCAKE_SPACE } from 'views/Voting/config'
+import { cakePoolBalanceStrategy, createTotalStrategy } from 'views/Voting/strategies'
```

Comment:
I believe namespace import * as should be able to tree shaking. 
it's not able to tree shaking only if we have object
```
export const strategies = {
 // 
}
```